packages/apputils/src/thememanager.ts

```diff
@@ -194,14 +198,22 @@ class ThemeManager {
    * Handle a load finished.
    */
   private _finishLoad(): void {
-    this._host.fit();
+    this._fitAll(this._host);
     this._loadPromise = null;
 
     if (this._pendingTheme) {
       this._loadTheme();
     }
   }
 
+  /**
+   * Fit a widget and all of its children, recursively.
+   */
+  private _fitAll(widget: Widget): void {
```

Comment:
I'd make this a free function, that way you dont have the allocation a new closure each time it's called.